k8s/rke2-mc/templates/rke2-calico-config.template.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.cattle.io/v1
```

Comment:
should be in the cloud-init of the servers.

Reply:
This file is provided as an example

Reply:
Since we can't provide full network config at init time I prefer to leave default and working network config and provide recommended steps to reconfigure it as required